packages/experimental-utils/typings/typescript.d.ts

```diff
@@ -0,0 +1,33 @@
+import 'typescript';
```

Comment:
do we still need to keep `packages/eslint-plugin/typings/typescript.d.ts` now that we've moved the utils here?

If we still need it - do we need *all* of it, or can we delete parts of it?

Reply:
removed unneeded parts